Mathlib/GroupTheory/Perm/Cycle/Basic.lean

```diff
@@ -1830,7 +1827,7 @@ open Equiv
 
 namespace List
 
-variable [DecidableEq α] {l : List α}
+variable [i4 : DecidableEq α] {l : List α}
```

Comment:
What's the motivation for this?

Reply:
It was a temporary change to see which `[DecidableEq α]` variable is still available. Then I forgot to change back.
```suggestion
variable [DecidableEq α] {l : List α}
```